DOWNSTREAM_OWNERS

```diff
@@ -1,11 +1,19 @@
 approvers:
   - Miciah
   - alebedev87
-  - arjunrn
   - thejasn
```

Comment:
You can remove thejasn too.  (Sadface.)

Reply:
Or put them under `emeritus_approvers:`.

Reply:
Removed.